packages/analyzer/src/features/framework-plugins/catalyst/catalyst.js

```diff
@@ -1,7 +1,11 @@
 import { attrDecoratorPlugin } from '../decorators/attr.js';
 import { controllerPlugin } from './controller.js';
+import { toKebabCase } from '../../../utils/index.js'
 
 export const catalystPlugin = () => [
-  attrDecoratorPlugin(),
+  attrDecoratorPlugin(attr => {
+    attr.name = `data-${toKebabCase(attr.name)}`;
```

Comment:
There is some nuance to how we change names that is a little more than kebabCase does, for example we special case acronyms so `srcURL` becomes `src-url` (as opposed to `src-u-r-l`), it may warrant bringing in catalyst as a dependency just to use the case transform; but otherwise this looks good.

Reply:
Thanks, that's definitely helpful, was wondering if there were any cases like these that I missed 😄. Would you be able to point me at the function in the source code of Catalyst that takes care of the attribute renaming? And is it publicly available/exported from Catalyst?

Reply:
Had to import it from `'@github/catalyst/lib/dasherize.js'`, but that works 🙂